src/korean.js

```diff
@@ -104,7 +104,7 @@ class Korean {
 
   doPast(word) {
     let presentWord = doPresent(word);
-    // stuff for past tense
+    return presentWord.concat('ㅆ어')
```

Comment:
@thisisrai, 
For next time: **Before** pushing you're branch, make sure to fetch and merge the latest code and rebase your branch.